services/common/config/rs-server.yaml

```diff
@@ -12,75 +12,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# THIS FILE WAS AUTOMATICALLY CREATED FROM:
+#  - rs-server/services/common/config/rs-server.template.yaml
+#  - rs-server/resources/sync_config_files.py
+# DON'T MODIFY IT DIRECTLY !
+
 external_data_sources:
-  # AUXIP. For now only one ADGS but in the future there will be another one
   adgs:
-    domain: mockup-auxip-adgs-svc.processing.svc.cluster.local
+    domain: mockup-station-adgs-svc.processing.svc.cluster.local
     service:
       name: auxip
       url: http://127.0.0.1:5000
     authentication:
       auth_type: oauth2
       token_url: http://127.0.0.1:5000/oauth2/token
       grant_type: password
-      scope:
+      scope: null
       username: test
       password: test
       client_id: client_id
       client_secret: client_secret
       authorization: Basic test
-
   adgs2:
-    domain: mockup-auxip-adgs-svc.processing.svc.cluster.local
+    domain: mockup-station-adgs2-svc.processing.svc.cluster.local
     service:
       name: auxip
       url: http://127.0.0.1:5001
     authentication:
       auth_type: oauth2
       token_url: http://127.0.0.1:5001/oauth2/token
       grant_type: password
-      scope:
+      scope: null
       username: test
       password: test
       client_id: client_id
       client_secret: client_secret
       authorization: Basic test
-
-  # CADIP stations. MPS has Authorization but no scope. MTI has scope but no Authorization
-  # the following one is for tests onlys
```

Comment:
@agrosu1978 @Padeanu the comment says that this is for tests only... but these values are used in production in the helm charts, is this normal ?